registry/handlers/app.go

```diff
@@ -67,6 +68,8 @@ type App struct {
 
 	redis *redis.Pool
 
+	trustKey libtrust.PrivateKey
```

Comment:
Please document correct use of this field: only for signing backwards compat requests.